update-issues/main.js

```diff
@@ -81,17 +85,21 @@ async function archiveOldIssues(issue) {
   }
 }
 
-async function unarchiveIssues(issue) {
-  // If the archive label is removed from an archived issue, unarchive it.
+async function unarchiveIssuesAndPRs(issue) {
```

Comment:
It's out of the scope of this change, but why is this method called `unarchiveIssuesAndPRs` if it is called on a single issue instead of an array of issues? Same question for the next few methods.

Reply:
`processIssues` (now `processIssuesAndPRs`) runs issues one at at time through each of the tasks in `ALL_ISSUE_TASKS` (now `ALL_TASKS`).  I feel the use of the plural in the task names still makes sense to me.